packages/ember-template-compiler/lib/plugins/transform-render-model.js

```diff
@@ -0,0 +1,54 @@
+import { deprecate } from 'ember-metal/debug';
```

Comment:
Small nit-pick, but this file doesn't transform anything.  Can you rename to `deprecate-render-model.js` instead?